project-base/kubernetes/deployments/smtp-server.yml

```diff
@@ -12,6 +12,9 @@ spec:
             containers:
             -   name: smtp-server
                 image: namshi/smtp:latest
+                env:
+                -   name: RELAY_NETWORKS
+                    value: :192.168.0.0/16:10.0.0.0/8:172.16.0.0/12
```

Comment:
What exactly do IP addresses mean? Are they somehow connected to our infrastructure or they are independent?

Reply:
allowing access from private networks that can docker containers run on